src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryListView.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from '@patternfly/react-core';
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+export type ConnectorTypesListItemProps = {
+  id: string;
+  labels: string[];
+  title: string;
+  verison: string;
+};
+
+export const ConnectorTypeListItem: React.FunctionComponent<ConnectorTypesListItemProps> =
+  ({ id, labels = [], title, verison }) => {
+    const [selectedDataListItemId, setSelectedDataListItemId] =
```

Comment:
We probably don't need these here, I suspect selection will be a concern that's passed into the gallery component.  I haven't fully thought it through mind you as we've also talked about having action buttons, so I've been holding off worrying about this aspect until those conversations seem resolved.